Media/DisplayMedia/Strategies/AbstractStrategy.php

```diff
@@ -58,8 +58,14 @@ public function displayPreview(MediaInterface $media)
      */
     public function displayMediaForWysiwyg(MediaInterface $media, $format = '')
     {
-        return '<img class="tinymce-media" src="' . $this->getFileUrl($media->getThumbnail()) . '" alt="'
-            . $media->getAlt($this->request->getLocale()) . '" data-id="' . $media->getId() . '" />';
+        return $this->container->get('templating')->render(
```

Comment:
To avoid the `abstractStrategy` `containerAware`, I would put the containerAware behaviour on the manager, and then the proxy method to render in the manager.

This way you could use the render method like in the DisplayBlockStrategies